format/view-spec.md

```diff
@@ -160,6 +179,57 @@ Each entry in `version-log` is a struct with the following fields:
 | _required_  | `timestamp-ms` | Timestamp when the view's `current-version-id` was updated (ms from epoch) |
 | _required_  | `version-id`   | ID that `current-version-id` was set to |
 
+#### Storage Table Identifier
+
+The table identifier for the storage table that stores the precomputed results.
+
+| Requirement | Field name     | Description |
+|-------------|----------------|-------------|
+| _optional_  | `catalog`      | A string specifying the name of the catalog. If set to `null`, the catalog is the same as the view's catalog |
```

Comment:
What are some reasons why the catalog and namespace should not be inherited from the MV's catalog and namespace?

IIUC the proposal allows:

- catalog1.ns1.source_table (source table)
- catalog2.ns2.mv (materialized view)
  - catalog3.ns3.storage_table (storage table)

Reply:
I agree with you that it's not a good idea to store the storage table in another catalog than the materialized view. My initial proposal didn't include the catalog. This would also get us around the issues with the "catalog alias" problem.

But I think some comments wanted to include the catalog. (I can't find it though)

I think having a dedicated namespace for all storage tables might be a use case for having a different namespace than the materialized view.

Reply:
Having a dedicated namespace makes sense. A separate catalog is what I can't think of a use for personally. If you can find the comments it would be helpful.

(I'm not asking for a change, as a person who has to possibly implement this downstream in engines I was thinking about use-cases).

Reply:
+1 on avoid a different `catalog` field that may introduce engine interoperability challenges that I personally feel quite unnecessary, unless there's a strong use case